src/test/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/EmailServiceTest.java

```diff
@@ -1,12 +1,14 @@
 package uk.gov.companieshouse.api.strikeoffobjections.service.impl;
 
+import org.junit.jupiter.api.BeforeEach;
```

Comment:
BeforeEach is showing up as unused in IDE

Reply:
Removed